src/webgl/p5.RendererGL.Immediate.js

```diff
@@ -45,6 +45,17 @@ const immediateBufferStrides = {
   uvs: 2
 };
 
+p5.RendererGL.prototype.beginContour = function() {
+  this.immediateMode.isContour = true;
+  this.immediateMode.isFirstContour = true;
+  this.immediateMode.geometry.contourVertices = [];
```

Comment:
Do you think we can store this on `this.immediateMode` instead of `this.immediateMode.geometry`? We use the same geometry class for 3D models loaded with `loadModel()`, which won't have this notion of contours, so this should probably just be internal to `p5.RendererGL`.